src/renderer/components/input/input_validators.ts

```diff
@@ -92,26 +116,24 @@ export const isExtensionNameInstall = inputValidator({
   validate: value => isExtensionNameInstallRegex.isMatch(value),
 });
 
-export const isPath = inputValidator<true>({
+export const isPath = asyncInputValidator({
   debounce: 100,
   condition: ({ type }) => type === "text",
   validate: async value => {
-    try {
-      await fse.pathExists(value);
-    } catch {
-      throw new AsyncInputValidationError(`${value} is not a valid file path`);
+    if (!await fse.pathExists(value)) {
+      throw new Error(`"${value}" is not a valid file path`);
     }
   },
 });
 
-export const minLength = inputValidator({
+export const minLength = inputValidator<true>({
```

Comment:
First time see this notation, does it mean same as `inputValidator<boolean>` or it can't be `false`?

Reply:
It means that the second parameter for the validate function is required. I can make that easier to read.

Reply:
I have backed out the ability to specify that it is required, thus making this easier to read.